twilight-http/src/routing.rs

```diff
@@ -2757,6 +2760,11 @@ impl Display for Route<'_> {
                     Display::fmt(limit, f)?;
                 }
 
+                if let Some(kind) = kind {
+                    f.write_str("&type=")?;
+                    Display::fmt(kind, f)?;
+                }
```

Comment:
Looking at this, this has the same issue that I spotted in #2341.
If you don't set a `after` this will give an error. This is probably something we should look into fixing all over the place, possible with some custom struct.

But for this pr we should just make some one off fix I guess?